### PR TITLE
added /C40/

### PR DESCRIPTION
**Pull request by viridispirus**
JaMopp restriction on Java versions source code.

### CODE REVIEW

**Comment by RomanLangrehr** [comment]
[PDF](http://viridispirus.github.io/Beagle/branches/Discussion/Requirements%20Specification.pdf)

File: doc/Requirements Specification/Parts/Discussion.lyx [hunk 1]
```diff
@@ -263,6 +263,20 @@ gls{PCM}
...
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/C40/"
+
+\end_inset
+
+ In the use of JaMopp the source code's Java version likely is restricted
+ by JaMopp.
+ The question is, if it is useful to further projects to get independent
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Discussion.lyx]: I think it should be "useful for further projects"

Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Discussion.lyx]: I don't think we can mention something with "The question is..." in the SRS.
We have to specify our software, not the problems with our software!!!

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Discussion.lyx]: > I don't think we can mention something with "The question is..." in the SRS.

Absolutely.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**